### ISSUE TITLE
Change loss by country query (GO)

### ISSUE THREAD

**Issue by d4weed** [Vizzuality/gfw#915]
Should be

WITH loss as (SELECT iso, SUM(loss) sum_loss               
FROM umd_nat               
WHERE thresh = 10               
GROUP BY iso) 
SELECT c.iso, c.name, c.enabled, sum_loss FROM loss, gfw2_countries c WHERE loss.iso = c.iso  AND NOT sum_loss = 0 ORDER BY sum_loss DESC LIMIT 10

**Comment by apercas** [comment]
Done.
